src/sdk/sql_cluster_router.cc

```diff
@@ -625,6 +625,31 @@ bool SQLClusterRouter::ShowDB(std::vector<std::string>* dbs, hybridse::sdk::Stat
     }
     return true;
 }
+ 
+std::vector<std::string> SQLClusterRouter::GetAllTables(hybridse::sdk::Status* status){
+
+    std::vector<std::string> dbs;
+    std::vector<std::string> all_table;
+    bool ok = SQLClusterRouter::ShowDB(&dbs, status);
+    if (!ok) {
+        return all_table;
+    }
+//     TODO: support getting all table names in one RPC
```

Comment:
remove this comment. Getting table names only is from the cache, not rpc.

Reply:
for we firstly call function `NsClient::ShowDatabase()` there should be request and response.

Reply:
updated.